### ISSUE TITLE
DRM problem? Download videos but turn out to be pictures

### ISSUE THREAD

**Issue by Thom1573** [Voldrix/onlyfans-dl-2#95]
One account that I subscribe only allows me to download the videos posted before June and from then on, any video posted was downloaded in jpg format by this scraper.

**Comment by PurplePocket** [comment]
Yeah, the DRM videos are new and don't seem to be found the same way other videos are. 
So far, no tool i could find has been able to find and download them.

Even the scummy $15 a month software can't seem able to download them. 
I can only pray for a breakthrough in piracy or leisure in the OF devs.

**Comment by BeachNL** [comment]
> The image is just the thumbnail, it is downloaded as a placeholder for when DRM is enabled on the video, which the script cannot download. Circumventing DRM is above my skill level unfortunately.

is there a place I can DM you on? Got some questions

**Comment by Voldrix** [comment]
@BeachNL same username on discord, or Voldrix_Suroku on reddit.
